admin/tabs/general/roles/RoleModel.ts

```diff
@@ -279,11 +324,12 @@ export class RoleModel extends HoistModel {
                     actions: [this.editAction()],
                     omit: this.readonly
                 },
-                {field: {name: 'name', type: 'string'}},
+                {field: {name: 'name', type: 'string'}, isTreeColumn: true},
                 {field: {name: 'category', type: 'string'}, hidden: true},
                 {field: {name: 'lastUpdated', type: 'date'}, ...Col.dateTime, hidden: true},
                 {field: {name: 'lastUpdatedBy', type: 'string'}, hidden: true},
-                {field: {name: 'notes', type: 'string'}, filterable: false, flex: 1}
+                {field: {name: 'notes', type: 'string'}, filterable: false, flex: 1},
+                {field: {name: 'isGroupRow', type: 'bool'}, hidden: true, excludeFromChooser: true}
```

Comment:
Does this need to be a hidden column? Seems like it could just be a `Store` `Field`?